pkg/onchain/storage/storage.go

```diff
@@ -55,19 +53,14 @@ func Get(client *node.Client, address string, key string) (map[string][]byte, er
 		return nil, fmt.Errorf("calling get_datastore_entries '%+v': %w", entries, err)
 	}
 
-	dataStore := ""
+	var dataStore []byte
```

Comment:
if you know the size of the object, you can directly set it using a make.

Reply:
The following code doesn't work ?
```suggestion
	dataStore := make([]byte, chunkNumber)
```

Reply:
No because in the `dataStore` variable we store as many bytes as each chunk contains.
If we use your code it means we're allocating 4 bytes if chunkNumber = 4, whereas really we need many more to store all the data contained in the 4 chunks of the website.